docs/autoloop.md

```diff
@@ -25,6 +25,10 @@ channel balance of 1M sats. Keep in mind that on first time use this will use
 the default budget parameters. If you wish to configure a custom budget you can
 find more info in the [Budget](#budget) section.
 
+In addition, if you want to control the maximum amount of fees you want to spend
```

Comment:
nit: "the maximum amount of fees you want to spend" => "the maximum amount of fees you're willing to spend".